modules/gmc-agent/src/main/java/org/gridgain/dto/tracing/Annotation.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gridgain.dto.tracing;
+
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+/**
+ * DTO for annotation.
+ */
+public class Annotation {
+    /** Timestamp. */
+    private long ts;
+
+    /** Value. */
+    private String val;
+
+    /**
+     * Default constructor.
+     */
+    public Annotation() {
+        // No-op.
+    }
+
+    /**
+     * @param ts Timestamp.
+     * @param val Value.
+     */
+    public Annotation(long ts, String val) {
+        this.ts = ts;
+        this.val = val;
+    }
+
+    /**
+     * @return Timestamp.
+     */
+    public long getTs() {
```

Comment:
Lets use "full" name for methods: `get(set)Timestamp()`

Reply:
Fix.